charts/c100-application/values.dev.template.yaml

```diff
@@ -30,4 +30,8 @@ redis:
     persistence:
       enabled: false
   auth:
-    enabled: false
\ No newline at end of file
+    enabled: false
+
+global:
+  enableKeyVaults: false
+  environment: dev
```

Comment:
don't check this in, it's set by jenkins automatically with the correct env

Reply:
Needed for the IF statement above - can remove if the IF is not needed.